docker-compose.yml

```diff
@@ -10,6 +11,7 @@ services:
     environment:
       SERVER_TYPE: 'WEB'
   bgio:
+    image: freeboardgames/gbio:latest
```

Comment:
gbio? Typo? Should be bgio?

Reply:
Yes, typo but was useless anyways. It is the same image as the web server just with a different variable for `SERVER_TYPE`.